lib/core/src/sdk.rs

```diff
@@ -278,6 +291,14 @@ impl LiquidSdk {
             .clone()
             .start(self.shutdown_receiver.clone())
             .await;
+        if let Err(err) = self
```

Comment:
A few comments here:
1. For consistency and also because the run method starts its own tokio task I think it is better to rename the run to start
2. If the start returns error then we shouldn't ignore it, I think it should fail the connect.
3. Since this method is called as part of connect it shouldn't block the caller. Right now as can be seen here: https://github.com/breez/breez-sdk-liquid/pull/568/files#diff-39d1d030470dc22070380fe8f700011c8f39961bbbf0245cf825c4bffedc1b15R59 the run invokes some network requests. I think we can insert all the blocking logic into the task loop.